### PR TITLE
Command line

### PR DESCRIPTION
**Pull request by filipecabaco**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM java:7
+
+RUN mkdir -p /tmp/bassethound
+RUN mkdir -p /tmp/sbt
+
+# Install sbt
+WORKDIR /tmp/sbt
+ADD http://dl.bintray.com/sbt/native-packages/sbt/0.13.11/sbt-0.13.11.tgz" .
```
Comment (RCM7) [inline Dockerfile]: Where is the opening quote? :stuck_out_tongue:

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+RUN tar -xvf /tmp/sbt/sbt-0.13.11.tgz
+RUN ls -al /tmp/sbt/sbt/bin
```
Comment (RCM7) [inline Dockerfile]: wut ? :D why would you need an ls ?

Comment (filipecabaco) [inline Dockerfile]: I was testing around... LOL

File: Dockerfile (hunk 1 above)
Comment (filipecabaco) [inline Dockerfile]: on the rest of the copy paste... damn...

**Pull request merged**